File: JavaBasic/129_inverted_triangle.java
// 129	129_inverted_triangle.java	Print an inverted right-triangle star pattern using nested loops.
public class _129_inverted_triangle {
    public static void main(String[] args) {
        int n = 5;
        for (int i = n; i >= 1; i--) {
            for (int j = 1; j <= i; j++) System.out.print("* ");
            System.out.println();
        }
    }
}

File: JSBASIC/129_inverted_triangle.js
// 129	129_inverted_triangle.js	Print an inverted right-triangle star pattern using nested loops.

let n = 5;
console.log("Inverted right-triangle pattern:");
for (let i = n; i >= 1; i--) {
    let row = "";
    for (let j = 1; j <= i; j++) {
        row += "* ";
    }
    console.log(row);
}
